src/trans/gpu/internal/trltom_mod.F90

```diff
@@ -94,7 +94,7 @@ SUBROUTINE TRLTOM(ALLOCATOR,HTRLTOM,PFBUF_IN,PFBUF,KF_FS)
     USE TPM_DISTR,              ONLY: D, NPRTRW, NPROC, MYPROC, MYSETW
     USE TPM_GEN,                ONLY: LSYNC_TRANS, NERR
 #if ECTRANS_HAVE_MPI
-    USE MPI_F08,                ONLY: MPI_COMM, MPI_FLOAT, MPI_DOUBLE, MPI_ALLTOALLV
```

Comment:
This might need __HIP_PLATFORM_AMD__ protection?

Reply:
I did consider that. Then I noticed that we don't explicitly import `MPI_ALLTOALLV` in `TRMTOL`, so it doesn't seem to be an issue.